crypto/doc/CHANGELOG_cc3xx_mbedcrypto.rst

```diff
@@ -38,7 +38,7 @@ Added a new build of nRF_cc3xx_mbedcrypto libraries for nRF9160, nRF52840, and n
   * :file:`crypto/nrf_cc312_mbedcrypto/lib/cortex-m33/**/libnrf_cc312_legacy_crypto_0.9.17.a`
   * :file:`crypto/nrf_cc312_mbedcrypto/lib/cortex-m33/**/libnrf_cc312_core_0.9.17.a`
 
-* nrf_cc310_mbedcrypto, nRF9160 variants
+* nrf_cc310_mbedcrypto, nRF91 variants
```

Comment:
```suggestion
* nrf_cc310_mbedcrypto, nRF91 Series variants
```

Regarding my comment above, I think the lowercase must've just been a typo, then.